app/js/components/cohorts/cohortTable.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes, Component } from 'react';
+
+
+
+class cohortTable extends Component {
```

Comment:
can we make this component stateless, it's not compulsory though just thinking out loud